build/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,13 @@
+$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$logMsi = Join-Path -Path $env:TEMP -ChildPath ("{0}-{1}-MsiInstall.log" -f $env:ChocolateyPackageName, $env:chocolateyPackageVersion)
+
+$packageArgs = @{
+    packageName    = $env:ChocolateyPackageName
+    fileType       = 'MSI'
+    silentArgs     = "/qn /norestart /l*v `"$logMsi`""
```

Comment:
l*v writes the verbose windows installer log to the file `$logMsi`

The verbose windows installer log can be quite big, and is never typically needed unless you're trying to troubleshoot something. 

Should we be enabling this?

Reply:
I've removed the verbose logging, we can re-enable at a later time if needed.